vm/vm.go

```diff
@@ -324,23 +331,23 @@ func (v *vm) Execute(txns []core.Transaction, declaredClasses []core.Class, paid
 
 	if context.err != "" {
 		if context.errTxnIndex >= 0 {
-			return nil, nil, nil, TransactionExecutionError{
+			return nil, nil, nil, context.executionSteps, TransactionExecutionError{
```

Comment:
that's also an error, I think we should get rid of `context.executionSteps` here

Reply:
I'm surprised this is here it's already removed in previous pushes,
might be a github thing.